pkg/apis/elasticsearch/v1/name.go

```diff
@@ -29,10 +29,10 @@ const (
 	statefulSetTransportCertificatesSecretSuffix = "transport-certs"
 
 	// calling this secret "xpack-file-realm" is conceptually wrong since it also holds the file-based roles which
-	// are not part of the file realm - let's still keep this legacy name for convenience
-	rolesAndFileRealmSecretSuffix = "xpack-file-realm"
+	// are not part of the file realm - let's still keep this legacy name for convenience.
+	rolesAndFileRealmSecretSuffix = "xpack-file-realm" // nolint:gosec
```

Comment:
I wonder if we want to disable these "hardcoded secret" checks entirely as I cannot imagine we really have any place where we might accidentally put in a hardcoded secret